content/docs/studio/user-guide/account-management.md

```diff
@@ -5,24 +5,59 @@ Iterative Studio. In the menu that comes up, click on `Profile`. Your user
 profile page will open. This page has multiple sections that are described
 below.
 
+## Profile
+
+Here, you can see your name and profile picture. If you signed up with a
+GitHub.com, GitLab.com or Bitbucket.org account, these details are fetched from
+your connected Git hosting account.
+
+You can edit your name.
+
 ## Account
 
-Here, you can see your avatar, user name and email address. These details are
-fetched from your connected Git account and cannot be changed.
+Here, you can see your username, password and email addresses. If you signed up
+with a GitHub.com, GitLab.com or Bitbucket.org account, the username and email
+address are fetched from your connected Git hosting account.
+
+You can update your username and password.
+
+**Managing email addresses:**
+
+You can add multiple email addresses to a single Studio account. You can login
+to the account with any of your verified email addresses.
```

Comment:
> You can add multiple email addresses to a single Studio account. You can login to the account with any of your verified email addresses.

May I ask why multiple addresses is allowed?

And furthermore why allowing login with any of them? And is that true even if the signup was via GH/GL/BB? (unclear from this doc)

Reply:
Multiple email addresses are allowed, just like in GitHub. The primary one can't be deleted; it's the one that gets account notifications. You could change your email address for a Studio account by adding a new one and setting that as primary and then deleting the old (non-primary) one.

If they are all my email addresses, they why not let me log in with any of them? Yes, it's true regardless of the signup method, as long as you have set a password in your account - have added this to the doc.

Reply:
> Multiple email addresses are allowed, just like in GitHub

Yes but why? Is there a use case? I'm just curious -- sounds like possibly unnecessary complication.

Reply:
Someone could use either of their email addresses to login (if they have multiple - such as for work and personal). It also lets them change the email address for their Studio account by adding the new one and deleting the old one (although for this, we could have added a `Change email` functionality). Also, I don't remember, but I think there was another (more technical) reason for this that applied to email signups. I can ask in the team if necessary.

Reply:
It's possible to lose access to email accounts for some reason and keeping multiple email addresses recorded in services (if possible) is usually a better approach. @jorgeorpinel